spec/rubocop/cop/layout/space_inside_block_braces_spec.rb

```diff
@@ -391,6 +391,23 @@
               }
           RUBY
         end
+
+        it 'accepts when braces are aligned in multiline block with bracket' do
+          expect_no_offenses(<<~RUBY)
+            foo {[
+              bar
+            ]}
+          RUBY
+        end
+
+        it 'registers an offense when braces are not aligned in multiline block with bracket' do
+          expect_offense(<<~RUBY)
+            foo {[
+              bar
+              ]}
+               ^{} Space inside } detected.
+          RUBY
```

Comment:
Can you add `expect_correction` code? It looks like `InfiniteCorrectionLoop` error is not resolved.

Reply:
Ops, I added it. Thank you.